README.md

```diff
@@ -69,9 +69,10 @@ Terminology
 
 Current build status
 ====================
+
 Linux: [![Circle CI](https://circleci.com/gh/conda-forge/thredds_crawler-feedstock.svg?style=svg)](https://circleci.com/gh/conda-forge/thredds_crawler-feedstock)
 OSX: [![TravisCI](https://travis-ci.org/conda-forge/thredds_crawler-feedstock.svg?branch=master)](https://travis-ci.org/conda-forge/thredds_crawler-feedstock) 
-Windows: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/conda-forge/thredds_crawler-feedstock?svg=True)](https://ci.appveyor.com/project/conda-forge/thredds_crawler-feedstock/branch/master)
+Windows: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/conda-forge/thredds-crawler-feedstock?svg=True)](https://ci.appveyor.com/project/conda-forge/thredds-crawler-feedstock/branch/master)
```

Comment:
So, the icon here will be screwy, but that can be fixed manually. Also, it should not be a problem in the next `conda-smithy` release. ( https://github.com/conda-forge/conda-smithy/pull/195 )